src/JustSaying/Messaging/MessageSerialization/IMessageSerialisationRegister.cs

```diff
@@ -9,20 +10,20 @@ public interface IMessageSerializationRegister
         /// </summary>
         /// <param name="body">Message must always have Subject and Message properties</param>
         /// <returns></returns>
-        Message DeserializeMessage(string body);
+        (Message, MessageAttributes) DeserializeMessage(string body);
```

Comment:
We shouldn't use tuples in the public surface.

Reply:
Have pulled this out to a separate `MessageWithAttributes` type